dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -131,6 +131,13 @@ public String formatByGroup() {
               .collect(Collectors.joining(", "));
 
       builder.append("Referenced by: ");
+
+      // A group (LinkageErrorCause) is marked unreachable if none of the errors in it is reachable
```

Comment:
Let's not change the dashboard report in this PR. I'm still not convinced this is something we should do.

Reply:
Removed the mark from the dashboard.